README.md

```diff
@@ -12,9 +12,10 @@
       src="https://cdn.rawgit.com/webpack/media/e7485eb2/logo/icon.svg">
   </a>
   <h1>UglifyJS Webpack Plugin</h1>
-	<p>This plugin uses <a href="https://github.com/mishoo/UglifyJS2/tree/v2.x">UglifyJS v2</a> to minify your JavaScript.<p>
```

Comment:
The issue is that this is not an HTML block anymore (because the unindented HTML before it gets deleted by the script fetching the readme). It is now a YAML codeblock, showcasing some HTML markup in the codeblock.
If you want that this stays a HTML block when rendering the website, you must ensure that the first line of HTML *in the rendered document* is unindented. And the webpack.js.org website removes the beginning of the readme (until the first `<h1>` it finds, so this `<p>` was becoming the first HTML line (but it is indented)